source/_static/css/custom.css

```diff
@@ -526,16 +508,17 @@ div[role=navigation][aria-label~=breadcrumbs] {
 .highlight-json .kc, .kd, .kn, .kp, .kr, .kt, .mf, .mh, .mi, .mo {
```

Comment:
This may be out of scope, but while we're here, I wonder if there's anyway we could tweak the json highlighting for the Swagger UI to be in-line with the other API pages?

Reply:
I was noticing the Swagger UI code blocks were hard to read (mainly the yellow JSON strings), but did confirm they were that way before as well and thus hadn't gotten worse -- so, I didn't pull on that thread. I'll see if there is anything quick

Reply:
I have a vague recollection from when I first implemented this that it was something of a hassle to adjust the highlighting, which probably explains why I left it as the default colors... You may have better luck than me though.